openexplorer.\_private\_tools.exceptions.BadCallError
=====================================================

.. currentmodule:: openexplorer._private_tools.exceptions

.. autoexception:: BadCallError